src/web/components/SharingPermission/BulkAddPermissions.tsx

```diff
@@ -42,70 +46,136 @@ export function BulkAddPermissions({
     },
   });
 
-  const { register, handleSubmit } = formMethods;
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { isDirty },
+  } = formMethods;
+
+  const watchPublisherChecked = watch('publisherChecked');
```

Comment:
Haven't used react-hook-form much before, so not sure if this is the best way to get the form values.

Reply:
Yeah - I'm still not sold on the pattern, but it's the way react-hook-form provides 🤷